src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,182 @@
+"use client";
+
+import React from "react";
+import { Svg, Document, Page, Text, View, StyleSheet, Path } from "@react-pdf/renderer";
+import { faFlag, faSquare, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+import { ParcelOfSpecificDateLocation } from "@/components/DayOverview/DayOverview";
+interface DayOverviewPdfProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateLocation[];
+}
+
+interface DayOverviewCardProps extends DayOverviewPdfProps {}
+
+interface CustomSVGProps {
+    icon: IconDefinition;
+    color: string;
+    fill: boolean;
+}
+
+const styles = StyleSheet.create({
+    page: {
+        display: "flex",
+        flexDirection: "column",
+        fontSize: "12px",
+    },
+    card: {
+        margin: "0px 40px 20px",
+    },
+    table: {
+        borderStyle: "solid",
+        borderWidth: 1,
+        borderRightWidth: 0,
+        borderBottomWidth: 0,
+    },
+    title: {
+        fontFamily: "Helvetica-Bold",
+        textTransform: "uppercase",
+        fontSize: "30px",
+        textAlign: "center",
+        margin: "30px",
+    },
+    dateLocationWrap: {
+        justifyContent: "space-between",
+        marginBottom: "5px",
+    },
+    svg: {
+        margin: "0 1px",
+        padding: "0 1px",
+    },
+    cell: {
+        borderStyle: "solid",
+        borderWidth: 1,
+        borderLeftWidth: 0,
+        borderTopWidth: 0,
+        padding: "3px",
+    },
+    row: { display: "flex", flexDirection: "row" },
+    bold: { fontFamily: "Helvetica-Bold" },
+
+    // cells
+    cell_logo: { flex: 1 },
+    cell_name: { flex: 4 },
+    cell_postcode: { flex: 4 },
+    cell_time: { flex: 2 },
+    cell_collection: { flex: 4 },
+    cell_instructions: { flex: 10 },
+});
+
+const dateTimeToAMPM = (datetime: string): string => {
+    return new Date(datetime).getHours() <= 11 ? "AM" : "PM";
+};
+
+const CustomSVG: React.FC<CustomSVGProps> = ({ icon, color, fill }) => (
+    <Svg width={12} height={12} viewBox="0 0 512 512" style={styles.svg}>
+        <Path
+            fill={fill ? color : undefined}
+            stroke={color}
+            strokeWidth={40}
+            d={icon.icon[4] as string}
+        />
+        {/*IconDefinition[4] is defined as string | string[] but in reality there is no use case*/}
+        {/*with string[] as svgPathData*/}
+    </Svg>
+);
+
+const DayOverviewDate: React.FC<{ date: Date }> = ({ date }) => {
+    const dateString = date.toLocaleDateString("sv-SE");
+    return (
+        <View style={styles.row}>
+            <Text style={styles.bold}>Date: </Text> <Text>{dateString}</Text>
+        </View>
+    );
+};
+
+const DayOverviewLocation: React.FC<{ location: string }> = ({ location }) => (
+    <View style={styles.row}>
+        <Text style={styles.bold}>Location: </Text> <Text>{location}</Text>
+    </View>
+);
+
+const DayOverviewHeader: React.FC<{}> = () => {
+    return (
+        <View style={styles.row}>
```

Comment:
Should be able to just put `styles.bold` in here instead of in every single view. (This is more what I meant when saying to set it in a higher level component)